cmd/image-builder/README.md

```diff
@@ -99,10 +100,13 @@ sign-config:
           type: token
 ```
 
-All enabled signers under `'*'` are used globally. Additionally, if a repository contains another signer configuration in the `org/repo` key, image-builder also uses this service to sign the image.
-If the job is running in CI (Prow), it picks up the current `org/repo` value from the default Prow variables. If binary is running outside of CI, `--repo` flag will have to be used. Otherwise, the configuration will not be used.
+All enabled signers under `'*'` are used globally. Additionally, if a repository contains another signer configuration
+in the `org/repo` key, image-builder also uses this service to sign the image.
+If the job is running in CI (Prow), it picks up the current `org/repo` value from the default Prow variables. If binary
+is running outside of CI, `--repo` flag will have to be used. Otherwise, the configuration will not be used.
```

Comment:
```suggestion
If the job runs in CI (Prow), it picks up the current `org/repo` value from the default Prow variables. If the binary
runs outside of CI, the `--repo` flag must be used. Otherwise, the configuration cannot be used.
```